docs/README.md

```diff
@@ -14,20 +14,32 @@ To contribute to the documentation, open a GitHub pull request.
 - You need to fork this repository.
 - Work on your fork's task branches, not the `master` branch.
 - Send pull requests to the `master` branch in this repository.
-- Run `make preview` to build the documentation and preview it locally at http://127.0.0.1:5500/.
 - If you want to submit a knowledge base article, see [Submitting a KB Article](#submitting-a-kb-article).
 
 The user documentation is written in reStructuredText (RST) - a plaintext markup language similar to Markdown. If you're not familiar with RST, see [ScyllaDB RST Examples](https://sphinx-theme.scylladb.com/stable/examples/index.html).
 
 The developer documentation is written in Markdown. See [Basic Markdown Syntax](https://www.markdownguide.org/basic-syntax/) for reference.
 
 
-## Prerequisites for Building User Documentation
+## Submitting a KB Article
+
+If you are submitting a Knowledgebase Article (KBA), use the following guidelines:
```

Comment:
It's not clear to me what "submitting" a KBA means here. At first I got the impression that this is some instructions on how to do something outside this repository... But this README is about this directory, so I think it will be clearer to title this section "The kb/ directory" and say something like:

"The kb/ directory holds knowledge-base articles that end up formatted in http://this-and-that/somewhere. To create a new knowledgebase article, make a copy ... etc."